dateutil/test/test_parser.py

```diff
@@ -536,9 +536,6 @@ def testMicrosecondPrecisionErrorReturns(self):
             dt = datetime(2008, 2, 27, 21, 26, 1, ms)
             self.assertEqual(parse(dt.isoformat()), dt)
 
-    # def testHighPrecisionSeconds(self):
```

Comment:
Hm... Not sure why this is commented out. If it fails it should be converted to an xfail.

Reply:
It should be deleted in the last commit. I missed it when I removed all the commented out tests. its here as a specific commit because I cherry picked two of the commits from the previously closed pr. see: https://github.com/dateutil/dateutil/pull/725/commits/42f62c2e659f617cc8153d38ecc8710cb9258bbd

Reply:
Oh you commented them out as you moved them over, I see.

Reply:
yeah, very scientific.